lavalink/client.py

```diff
@@ -51,16 +51,19 @@ def __init__(self, user_id: int, shard_count: int = 1,
         self.node_manager = NodeManager(self, regions)
         self.players = PlayerManager(self, player)
 
-        self._event_hooks = []
+        self._event_hooks = {}
 
         self._session = aiohttp.ClientSession(
             connector=aiohttp.TCPConnector(loop=loop),
             timeout=aiohttp.ClientTimeout(total=30)
         )  # This session will be used for websocket and http requests.
 
     def add_event_hook(self, hook):
-        if hook not in self._event_hooks:
-            self._event_hooks.append(hook)
+        if hook not in self._event_hooks.get("Unknown Event"):
```

Comment:
If I understand this correctly, what if `Unknown Event` doesn't exist? You can't do `if hook in None`

Reply:
Changed it in another commit